com.unity.testing.urp/package.json

```diff
@@ -0,0 +1,13 @@
+{
+	"name": "com.unity.testing.urp",
+	"displayName":"URP Graphic Tests Library",
+	"version": "1.0.0",
+	"unity": "2019.3",
+	"unityRelease": "0b4",
+	"description": "Common assets library for URP test projects.",
+	"keywords": ["qa", "test", "testing", "tests", "graphics", "HDRP"],
+	"dependencies": {
+	"com.unity.testframework.graphics": "7.8.7-preview",
```

Comment:
The latest gfx test framework is 7.8.8-preview

Reply:
Ok. I will update that, it must have been bumped after I branched off